apps/api/src/domain/models/song/song.entity.ts

```diff
@@ -62,7 +62,7 @@ export class Song extends Resource implements ITimeBoundable {
 
         this.titleEnglish = titleEnglish;
 
-        this.contributorAndRoles = contributorAndRoles.map(
+        this.contributorAndRoles = (contributorAndRoles || []).map(
```

Comment:
No- it would give an empty array. Essentially what is happening is that we are defaulting this to an empty array in case it is not defined. This should only happen in the validation context.